gitpg/carlcaceres/techinfo.txt

```diff
@@ -1,6 +1,10 @@
 1. c, 2. c++, 3. c#, 4. Java y 5. Python
 1. Visual Studio Code, 2. Sublime Text, 3. Netbeans y 4. Atom
 Linux Ubuntu y Windows
+<<<<<<< HEAD
```

Comment:
Hi @dkmar11 , this one still needs to be fixed

Reply:
Hi @J0c4, it was fixed the issue, i am going to have more care the next time. thanks a lot!